modules/apps/bookmarks/bookmarks-item-selector-web/src/main/java/com/liferay/bookmarks/item/selector/web/internal/item/selector/BookmarksEntryItemSelectorView.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.bookmarks.item.selector.web.internal.item.selector;
+
+import com.liferay.bookmarks.item.selector.web.internal.display.context.BookmarkEntriesItemSelectorDisplayContext;
+import com.liferay.bookmarks.model.BookmarksEntry;
+import com.liferay.info.item.selector.InfoItemSelectorView;
+import com.liferay.item.selector.ItemSelectorReturnType;
+import com.liferay.item.selector.ItemSelectorView;
+import com.liferay.item.selector.criteria.InfoItemItemSelectorReturnType;
+import com.liferay.item.selector.criteria.info.item.criterion.InfoItemItemSelectorCriterion;
+import com.liferay.portal.kernel.language.Language;
+import com.liferay.portal.kernel.theme.ThemeDisplay;
+
+import java.io.IOException;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Locale;
+
+import javax.portlet.PortletURL;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletContext;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Yang Cao
+ */
+@Component(
+	enabled = false, property = "item.selector.view.order:Integer=200",
+	service = ItemSelectorView.class
+)
+public class BookmarksEntryItemSelectorView
+	implements InfoItemSelectorView,
+			   ItemSelectorView<InfoItemItemSelectorCriterion> {
+
+	@Override
+	public String getClassName() {
+		return BookmarksEntry.class.getName();
+	}
+
+	@Override
+	public Class<InfoItemItemSelectorCriterion>
+		getItemSelectorCriterionClass() {
+
+		return InfoItemItemSelectorCriterion.class;
+	}
+
+	public ServletContext getServletContext() {
+		return _servletContext;
+	}
```

Comment:
Why is this method public? I don't see any usages outside of this class. If there are none, please remove it.